ext/opentelemetry-ext-opentracing-shim/src/opentelemetry/ext/opentracing_shim/__init__.py

```diff
@@ -77,6 +77,14 @@ def log_kv(self, key_values, timestamp=None):
         self._otel_span.add_event(event_name, event_timestamp, key_values)
         return self
 
+    def log(self, **kwargs):
+        logger.warning("Calling deprecated method log")
```

Comment:
Good reminder that we should consider [`deprecated`](https://pypi.org/project/Deprecated/) for this kind of thing.

Reply:
Can do, you want me to add Deprecated as an install dependency of the shim?

Reply:
Sorry, I didn't mean to suggest any action in this PR, just a reminder that we might want to do this in the future.

We talked about adding the library earlier but didn't yet have a good use case for it. This might be a good use case.